scripts/transforms/README.md

```diff
@@ -1,16 +1,18 @@
 ## Codemods
 
-These codemods should be ran with [`jscodeshift`](https://github.com/facebook/jscodeshift).
+These codemods (code transformers) can be ran with `npx`. If you installed the `instantsearch.js` package already with `yarn`, you can run them like this: `yarn instantsearch-scripts`).
 
-### addWidget / removeWidget
+### `addWidget-to-addWidgetss`
```

Comment:
there is a typo here, no?

Reply:
the title is what it does, not the name of the transform

Reply:
```suggestion
### `addWidget-to-addWidgets`
```